R/doAzureParallel.R

```diff
@@ -194,6 +194,8 @@ setHttpTraffic <- function(value = FALSE) {
   assign("expr", expr, .doAzureBatchGlobals)
   assign("exportenv", exportenv, .doAzureBatchGlobals)
   assign("packages", obj$packages, .doAzureBatchGlobals)
+  assign("github", eval(obj$github), .doAzureBatchGlobals)
+  assign("bioconductor", eval(obj$bioconductor), .doAzureBatchGlobals)
```

Comment:
Please update the docs for both of these features.

Reply:
updated the doc